ci/prow/config.yaml

```diff
@@ -51,6 +51,19 @@ deck:
       "artifacts/junit.*\\.xml":
       - "junit"
     announcement: "The old job viewer, Gubernator, has been deprecated in favour of this page, Spyglass.{{if .ArtifactPath}} For now, the old page is <a href='https://gubernator.knative.dev/build/{{.ArtifactPath}}'>still available</a>.{{end}} Please send feedback to Knative productivity."
+    lenses:
+    - lens:
+        name: build-log
+        config:
+          highlight_regexes:
+          - 'panic: test timed out after'
+          - timed out
```

Comment:
I wonder why this entry doesn't capture the one above it. Could it be the missing single quotes?

Reply:
It can capture it, but it's very easy to be ignored, see the screenshot below:
![image](https://user-images.githubusercontent.com/19399934/66964723-46e2b280-f02c-11e9-980a-1ee897a83320.png)

I believe if the whole sentence is highlighted, it will be more noticeable to the developers.

Reply:
Modified a bit to highlight the whole line.